src/core/UniswapV3Fork.sol

```diff
@@ -286,13 +284,12 @@ abstract contract UniswapV3Fork is SettlerAbstract {
         // )))
         bytes32 salt;
         assembly ("memory-safe") {
-            token0 := and(ADDRESS_MASK, token0)
-            token1 := and(ADDRESS_MASK, token1)
             poolId := and(UINT24_MASK, poolId)
             let ptr := mload(0x40)
-            mstore(0x00, token0)
-            mstore(0x20, token1)
             mstore(0x40, poolId)
+            mstore(0x20, token1)
```

Comment:
no masking it anymore?

Reply:
`token1` is masked by shifting `token0` up and then offsetting the address where we store `token1`. It costs 1 more gas, but it saves a bunch of contract size